docs/view-default.md

```diff
@@ -0,0 +1,169 @@
+
+![](./assets/view-default-simulated.png)
+
+```typescript
+const nodes = [
+  { id: 0, label: "Node A" },
+  { id: 1, label: "Node B" },
+  { id: 2, label: "Node C" },
+];
+const edges = [
+  { id: 0, start: 0, end: 0, label: "Edge Q" },
+  { id: 1, start: 0, end: 1, label: "Edge W" },
+  { id: 2, start: 0, end: 2, label: "Edge E" },
+  { id: 3, start: 1, end: 2, label: "Edge T" },
+  { id: 4, start: 2, end: 2, label: "Edge Y" },
+  { id: 5, start: 0, end: 1, label: "Edge V" },
+];
+
+const orb = new Orb.Orb(container);
+
+// Initialize nodes and edges
+orb.data.setup({ nodes, edges });
+
+// Render and recenter the view
+orb.view.render(() => {
+  orb.view.recenter();
+});
+```
+
+### Fixed node positions
+
+If you want to assign specific coordinates to your graph, you can do this by providing the `getPosition()` callback function.
+You can use this function to indicate which properties of your original nodes will be in the returned `IPosition` object (`{ x: number, y: number }` ) that allows Orb to position the nodes.
+
+![](./assets/view-default-fixed.png)
+
+```typescript
+const container = document.getElementById("graph");
+
+const nodes = [
+  { id: 0, label: "Node A", posX: -100, posY: 0 },
+  { id: 1, label: "Node B", posX: 100, posY: 0 },
+  { id: 2, label: "Node C", posX: 0, posY: 100 },
+];
+const edges = [
+  { id: 0, start: 0, end: 0, label: "Edge Q" },
+  { id: 1, start: 0, end: 1, label: "Edge W" },
+  { id: 2, start: 0, end: 2, label: "Edge E" },
+  { id: 3, start: 1, end: 2, label: "Edge T" },
+  { id: 4, start: 2, end: 2, label: "Edge Y" },
+  { id: 5, start: 0, end: 1, label: "Edge V" },
+];
+
+const orb = new Orb.Orb(container);
+orb.setView(
+  (context) =>
+    new Orb.DefaultView(context, {
+      getPosition: (node) => ({ x: node.data.posX, y: node.data.posY }),
+    })
+);
+
+// Initialize nodes and edges
+orb.data.setup({ nodes, edges });
+
+// Render and recenter the view
+orb.view.render(() => {
+  orb.view.recenter();
+});
+```
+
+### Property `getPosition`
+
+You can use this callback function to assign fixed coordinates to your nodes.
+
+The function has a node input (`INode`) which represents the Orb node data structure.
+You can access your original properties through `.data` property.
+There you can find all properties of your nodes that you assigned in the `orb.data.setup()` function.
+
+Here you can use your original properties to indicate which ones represent your node coordinates. (`node.data.posX`, `node.data.posY`)
+All you have to do is return a `IPosition` that requires 2 basic properties: `x` and `y`. (`{ x: node.data.posX, y: node.data.posY }`)
+
+### Property `render`
+
+Optional property `render` has several rendering options that you can tweak. Read more about them
+on [Styling guide](./styles.md).
+
+### Property `simulation`
+
+Fine-grained D3 simulation engine settings. This may be condensed into fewer, more abstract settings in the future.
```

Comment:
It would be great to, at least, list the properties that can be updated within `simulation` and a link to D3 documentation to read more about it. The user, if not using Typescript, needs to get into the code to figure out all the properties that can be set up in `simulation` settings.

Reply:
I thought about this, but we were talking about maybe abstracting this stuff away into a single property, so it didn't make sense for me to do work that will be deleted.